modules/microshift-greenboot-check-boot.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-boot_{context}"]
+= Checking an update's success with greenboot
+
+After an update, you can determine whether the update was declared valid by checking the boot. Such a health check can help you determine if you the system is working properly and you can proceed without a rollback.
+
+The health check script for updates is installed into the `/etc/greenboot/check/required.d` directory, and is automatically executed during the system boot. Exiting the script with a non-zero status will have the boot declared as failed.
+
+[IMPORTANT]
+====
+Wait until after an update boot is declared successful before starting third-party workloads. If a rollback is performed after workloads start, you can lose data. Some third-party workloads create or update data on a device before an update is complete. Upon rollback, the file system reverts to its state before the update.
+====
+
+.Procedure
+
+* Run the following command to check that an update is declared valid:
++
+[source, terminal]
+----
+$ sudo grub2-editenv - list | grep ^boot_success
+boot_success=1
+----
+
+//Let's give an example output of a successful boot. Is it just something like:
+.Example output for a successful boot
+
+[source, terminal]
+----
+$ boot_success=1
+----
```

Comment:
The example output is already in the first command. It's "boot_success=1" without the $ sign